toolkits/global/packages/global-author-list/README.md

```diff
@@ -54,18 +59,76 @@ Overrides the default ` & ` with a `...` before the final item in the list.
 **Example output**
 `Author 1, Author 2 ... Author 5`
 
-### Heads Up!
+### Enhance with a toggle button which expands/truncates the list
+
+#### Markup
+
+```html
+<div data-author-list>
+    <h3 data-author-list-heading>Authors</h3>
```

Comment:
If the heading is here for accessibility reasons, should it not be included for the non-js example as well?